src/content/pt/2019/compression.md

```diff
@@ -316,18 +316,18 @@ Adicionalmente, a percentagem de compressão Brotli é mais alta para conteúdo
 
 ## Como identificar oportunidades para compressão
 
-A ferramenta [Lighthouse](https://developers.google.com/web/tools/lighthouse) da Google permite aos utilizadores correrem uma série de auditorias contra páginas web. A [auditoria de compressão de texto](https://developers.google.com/web/tools/lighthouse/audits/text-compression) avalia se o site pode beneficiar de compressão baseada em texto adicional. Isso é feito através duma tentativa de compressão dos recursos avaliando se o tamanho dos objectos pode ser reduzido pelo menos 10% e 1400 bytes. Dependendo da pontuação, é possível ver uma recomendação para compressão nos resultados, com uma lista dos recursos específicos que podem ser comprimidos. 
+A ferramenta [Lighthouse](https://developers.google.com/web/tools/lighthouse) da Google permite aos usuários executarem uma série de auditorias para páginas web. A [auditoria de compressão de texto](https://developers.google.com/web/tools/lighthouse/audits/text-compression) avalia se o site pode beneficiar de compressão baseada em texto adicional. Isso é feito através de uma tentativa de compressão dos recursos avaliando se o tamanho dos objetos pode ser reduzido pelo menos 10% e 1400 bytes. Dependendo da pontuação, é possível ver uma recomendação para compressão nos resultados, com uma lista dos recursos específicos que podem ser comprimidos. 
 
 {{ figure_markup(
   image="ch15_fig8_lighthouse.jpg",
   caption="Sugestões de compressão do Lighthouse.",
-  description="A captura dum relatório do Lighthouse que mostra a lista de recursos (com respectivos nomes) e o tamanho do potencial ganho. Para o primeiro elemento há um ganho potencial significativo de 91 KB para 73 KB, enquanto que para outros ficheiros mais pequenos de 6 KB ou menos, encontram-se ganhos menores de 4 KB para 1 KB.",
```

Comment:
Hi @soulcorrosion , it was my mistake, but the `respectivos` word is correct here (not `respetivos`). It is the last correction that I would like suggest.

Reply:
All good. Done